app/views/facility_user_reservations/index.html.haml

```diff
@@ -41,8 +41,11 @@
           %td= order_detail.reservation
 
           %td.user-order-detail.order-detail-description
-            = link_to order_detail.description_as_html_with_facility_prefix,
-              facility_instrument_path(order_detail.facility, order_detail.product)
+            - if current_facility.try(:single_facility?)
```

Comment:
We probably should revisit this: the facility user orders partial just shows a text description with no link. The orders partial is shared in user and facility facing views though, where the reservation equivalents are not shared.